src/app/routes/getInitialData/index.js

```diff
@@ -18,7 +20,47 @@ const baseUrl = onClient()
   ? getBaseUrl(window.location.origin)
   : process.env.SIMORGH_BASE_URL;
 
-const getUrl = pathname => `${baseUrl}${pathname.replace(ampRegex, '')}.json`;
+export const validateRendererEnvironment = environment => {
+  const validRendererEnvironments = ['test', 'live'];
+
+  if (
+    !(
+      environment &&
+      validRendererEnvironments.includes(environment.toLowerCase())
+    )
+  ) {
+    logger.warn(
+      `Invalid parameter value [${environment}]. Usage: renderer_env=${validRendererEnvironments.join(
+        '|',
+      )}`,
+    );
+  }
+};
+
+const getPathFromUrl = url => {
+  return url.split(/[?]/)[0];
```

Comment:
I think there's a global API for this sort of thing e.g.

```js
const url = new URL('https://developer.mozilla.org/en-US/docs/Web/API/URL')
return url.pathname // '/en-US/docs/Web/API/URL'
```

Reply:
Using url-parse (already installed) works when I don't have the full URL